src/registry/registry.ts

```diff
@@ -10,6 +10,7 @@ import Fieldset from './fieldset';
 import PhoneNumber from './phoneNumber';
 import RadioField from './radio';
 import Selectboxes from './selectboxes';
+import textarea from './textarea';
```

Comment:
```suggestion
import TextArea from './textarea';
```
let's apply consistent casing